documentation/app/templates/components/list-item-template.hbs

```diff
@@ -0,0 +1,856 @@
+        <I.Link @section="#large" @depth={{2}}>Large (default)</I.Link>
+        <I.Link @section="#small" @depth={{2}}>Small (default)</I.Link>
+      </InPageNav>
+    </div>
+  </T.Panel>
+  <T.Panel>
+    <div class="doc-Page">
+      <div class="doc-Page-content">
+        <h2 id="type">Type</h2>
+        <div class="doc-list-item-template-types">
+          <p>Large <br />(Desktop)</p>
+          <Cut::ListItem::Template as |T|>
+            <T.Label as |Label|>
+              <Label.Icon @name="hexagon" />
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label>
+            <T.Badge @icon="hexagon" @text="Default" />
+            <T.Label2 as |Label|>
+              <Label.Icon @name="hexagon" />
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label2>
+          </Cut::ListItem::Template>
+          <p>Small<br />(Mobile)</p>
+          <Cut::ListItem::Template
+            class="doc-list-item-template-types_small"
+            as |T|
+          >
+            <T.Label as |Label|>
+              <Label.Icon @name="hexagon" />
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label>
+            <T.Badge @icon="hexagon" @text="Default" />
+            <T.Label2 as |Label|>
+              <Label.Icon @name="hexagon" />
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label2>
+          </Cut::ListItem::Template>
+        </div>
+        <h2 id="component-anatomy">Component anatomy</h2>
+        <p>The List Item Template is made up from nested components. Below is
+          the anatomy for the individual components. For more details on their
+          respective APIs, view the component API section under the code tab.</p>
+
+        <h3 id="section">Label</h3>
+        <p>The List Item Template Label is the primary part of the template. It
+          consists of a title, as well as an option icon for the title and a
+          section to place metadata like badges, labels, etc. There are two
+          available Labels in the List Item Template. The first is required and
+          is available via the
```

Comment:
Update copy to: 
"The List Item Template Label is the primary part of the template. It consists of a title, and an optional icon for the title, as well as an area to place metadata like badges, labels, etc. There are two available Labels in the List Item Template. The first is required and is available via the Label contextual component. The second Label is optional and can be used via the Label2 contextual component."

Reply:
i know this will soon change to use "Section" instead of "label" so feel free to ignore until we update.